src/qt/rpcconsole.cpp

```diff
@@ -798,6 +798,12 @@ void RPCConsole::removeWallet(WalletModel * const walletModel)
         ui->WalletSelectorLabel->setVisible(false);
     }
 }
+
+void RPCConsole::SetCurrentWallet(WalletModel* const wallet_model)
```

Comment:
Why `const`? The parameter of all connected signals is `WalletModel* wallet_model`, right?

Reply:
It's the pointer that's `const` here. Seems pretty reasonable, though it only has a net effect of preventing accidentally changing `wallet_model` in this method.